### DISCUSSION TITLE
Yolov8 using Linux Debian 9?

### DISCUSSION THREAD

**Discussion by rob-spotter** [ultralytics/ultralytics#7349] [Q&A]
Can Yolov8 Ultralytics be installed and run on Linux Debian OS 9?

Thanks.

**Comment by pderrenger** [comment]
@rob-spotter yes, YOLOv8 can be installed and run on Linux Debian 9. Ensure you have Python 3.6 or later and PyTorch 1.7 or later installed. Follow the installation instructions in our documentation, and you should be able to use YOLOv8 on your Debian system. If you encounter any issues, feel free to open an issue on the repo with details, and we'll be happy to assist! 🐍🔥
